kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js

```diff
@@ -22,19 +22,8 @@ define([
         let places, parent, container, inputControl;
 
         try {
-            inputControl = inputControlFactory.make({
-                bus: bus,
-                paramsChannelName: config.paramsChannelName,
-                channelName: bus.channelName,
-                initialValue: config.initialValue,
```

Comment:
Most of this is copy-pasted from the config object, so I just extended them.
This also allows any input-widget-specific things (like `availableValues` and `disabledValues` in this case) to just freely pass in.